manifests/base/server/argocd-server-service.yaml

```diff
@@ -8,11 +8,11 @@ metadata:
   name: argocd-server
 spec:
   ports:
-  - name: http
+  - name: tcp-argocdserver
     protocol: TCP
     port: 80
     targetPort: 8080
-  - name: https
+  - name: tcp-argocdservertls
     protocol: TCP
     port: 443
     targetPort: 8080
```

Comment:
Please note the ideally we would remove port 80 and keep only 443. In that case we could use https as protocol (istio does not support multiple protocols for a single service ports and as such we have to go down to TCP which works fine with two ports but we will lack http level information in Istio telemetry).
I did not want to put that into the PR as that would be probably too big change... or?

Reply:
We can't remove port 80 from this service, because many people actually run ArgoCD using HTTP mode and doing TLS termination elsewhere.

Reply:
Yes, that is my plan as well (doing TLS termination at the Istio gateway level) but to be honest, I believe we should have one way of doing it as default (like keep 443 as that is a simple, "secure" setup) and document all the other possibilities (like how to do TLS termination elsewhere).
I have Istio config for both (Istio passthrough with TLS termination at argocd-server and Istio TLS termination with Istio mTLS and argocd-server using http) and planning to send a PR to your Ingress docu page.